packages/core/src/codewhisperer/service/transformByQ/transformProjectValidationHandler.ts

```diff
@@ -170,7 +170,7 @@ export async function validateOpenProjects(
             // TODO: remove deprecated metric once BI started using new metrics
```

Comment:
Is this metric still planned for removal?

Reply:
Yes, but not yet. The others i removed because we have access to API latency / errors from our backend anyway, so we don't need to wait for our BI to get updated